src/App.tsx

```diff
@@ -66,14 +76,16 @@ function App(): ReactElement {
   });
 
   // Recording UI
-  const [imagePrefix, setImagePrefix] = useState<null | string>(null);
-  const [startAtFirstFrame, setStartAtFirstFrame] = useState(false);
+  // const [imagePrefix, setImagePrefix] = useState<null | string>(null);
+  // const [startAtFirstFrame, setStartAtFirstFrame] = useState(false);
```

Comment:
Temporarily disabling the RecordingControls info until I can reimplement it in the next update